R/tokens.R

```diff
@@ -465,7 +478,7 @@ compile_removals_regex <- function(remove_separators = FALSE,
     regex_to_remove <- removing_msg <- character()
 
     if (remove_separators) {
-        regex_to_remove <- c(regex_to_remove, "^[\\p{Z}\\p{C}]$")
+    #    regex_to_remove <- c(regex_to_remove, "^[\\p{Z}\\p{C}]$")
         removing_msg <- c(removing_msg, "separators")
```

Comment:
adding the message but not adding this to the regex restores the message whose absence was causing test failures.